ingestors/interfaces.go

```diff
@@ -0,0 +1,25 @@
+package ingestors
+
+import (
+	"time"
+
+	"github.com/librariesio/depper/data"
+)
+
+type Ingestor interface {
```

Comment:
Interface files are starting to get out of hand, this felt like a decent place to stash a bunch of small interfaces used by the package.